tests/unittests/test_security_install.py

```diff
@@ -78,36 +78,108 @@ def test_group_install(self):
     @pytest.mark.unit
     def test_group_dynamic_member_install(self):
         """Check update dynamic member in security group"""
-        self._common_install(
-            "some_id", group_dynamic_member.create,
+        self._common_install_read_and_update(
+            "security_group_id", group_dynamic_member.create,
             {'dynamic_member': {
                 "dynamic_set": "dynamic_set",
                 "security_group_id": "security_group_id"
-            }}
+            }},
+            ['secGroupObject'],
+            {'uri_parameters': {'objectId': 'security_group_id'}},
+            {
+                'status': 204,
+                'body': {
+                    'securitygroup': {
+                        'dynamicMemberDefinition': {
+                            'dynamicSet': 'oldDynamicSet',
+                        }
+                    }
+                }
+            },
+            # for update need to use 'secGroupBulk'
+            ['secGroupBulk'],
+            {
+                'request_body_dict': {
+                    'securitygroup': {
+                        'dynamicMemberDefinition': {
+                            'dynamicSet': 'dynamic_set'
+                        }
+                    }
+                },
+                'uri_parameters': {
+                    'scopeId': 'security_group_id'
+                }
+            },
+            {
+                'status': 204
+            }
         )
 
     @pytest.mark.internal
     @pytest.mark.unit
     def test_group_exclude_member_install(self):
         """Check insert member to exclude list in security group"""
-        self._common_install(
-            "some_id", group_exclude_member.create,
+        self._common_install_read_and_update(
+            "security_group_id|objectId", group_exclude_member.create,
             {'group_exclude_member': {
                 "objectId": "objectId",
                 "security_group_id": "security_group_id"
-            }}
+            }},
```

Comment:
This and the previous one look like they could probably be refactored to get a method that would return either of them since they look mostly the same?
It might be helpful if any further tests ended up being added for these things.